src/test/scala/org/hammerlab/guacamole/commands/VariantSupportSuite.scala

```diff
@@ -14,99 +15,111 @@ class VariantSupportSuite extends GuacFunSuite with Matchers with TableDrivenPro
-      heading = ("contig", "locus", "alleleCounts"),
-      variantAlleleLoci: _*
-    )
-
-    forAll(variantAlleleLociTable) {
-      (contig: String, locus: Long, alleleCountMap) =>
-        {
-          window.setCurrentLocus(locus)
-          val pileup = Pileup(window.currentRegions(), contig, locus, referenceContigSequence = grch37Reference.getContig(contig))
-          assertAlleleCounts(pileup, alleleCountMap)
-        }
+                       variantAlleleLoci: (String, Int, Seq[(String, String, Int)])*) = {
+    for {
+      (contig, locus, alleleCounts) ← variantAlleleLoci
+    } {
+      withClue(s"$contig:$locus") {
+
+        window.setCurrentLocus(locus)
+
+        val pileup =
+          Pileup(
+            window.currentRegions(),
+            contig,
+            locus,
+            referenceContigSequence = grch37Reference.getContig(contig)
+          )
+
+        assertAlleleCounts(pileup, alleleCounts: _*)
+      }
     }
   }
 
-  def assertAlleleCounts(pileup: Pileup,
-                         alleleCountMap: Map[String, Int]) = {
-    val computedAlleleCounts = VariantSupport.Caller.pileupToAlleleCounts(pileup)
-    computedAlleleCounts.size should be(alleleCountMap.size)
+  def assertAlleleCounts(pileup: Pileup, alleleCounts: (String, String, Int)*): Unit = {
+    val computedAlleleCounts =
+      (for {
+        AlleleCount(_, _, _, ref, alternate, count) ← VariantSupport.Caller.pileupToAlleleCounts(pileup)
+      } yield (ref, alternate, count)
+      )
+        .toArray
+        .sortBy(x ⇒ x)
 
-    for (alleleCount <- computedAlleleCounts) {
-      alleleCount.count should be(alleleCountMap(alleleCount.alternate))
-    }
+    computedAlleleCounts should be(alleleCounts.sortBy(x ⇒ x))
   }
 
-  lazy val gatkReads = TestUtil.loadReads(
-    sc,
-    "gatk_mini_bundle_extract.bam",
-    Read.InputFilters(mapped = true, nonDuplicate = false),
-    reference = grch37Reference).mappedReads.collect().sortBy(_.start)
-
-  lazy val nonDuplicateGatkReads = TestUtil.loadReads(
-    sc,
-    "gatk_mini_bundle_extract.bam",
-    Read.InputFilters(mapped = true, nonDuplicate = true),
-    reference = grch37Reference
-  ).mappedReads.collect().sortBy(_.start)
-
-  lazy val RnaReads = TestUtil.loadReads(
-    sc,
-    "rna_chr17_41244936.sam",
-    reference = grch37Reference).mappedReads.collect().sortBy(_.start)
+  def gatkReads(loci: String) =
+    TestUtil.loadReads(
+      sc,
+      "gatk_mini_bundle_extract.bam",
+      Read.InputFilters(
+        mapped = true,
+        nonDuplicate = false,
+        overlapsLoci = Some(LociSet.parse(loci))
+      ),
+      reference = grch37Reference
+    ).mappedReads.collect().sortBy(_.start)
+
+  def nonDuplicateGatkReads(loci: String) =
+    TestUtil.loadReads(
+      sc,
+      "gatk_mini_bundle_extract.bam",
+      Read.InputFilters(
+        mapped = true,
+        nonDuplicate = true,
+        overlapsLoci = Some(LociSet.parse(loci))
+      ),
+      reference = grch37Reference
```

Comment:
#401 didn't remove this `reference`; #415 does but it's not merged yet, so I'll leave it here and resolve conflicts as they arise